OrbitGl/SamplingReport.h

```diff
@@ -25,14 +25,14 @@ class SamplingReport
     void DecrementCallstackIndex();
     std::wstring GetSelectedCallstackString();
     void SetUiRefreshFunc( std::function<void()> a_Func ){ m_UiRefreshFunc = a_Func; }
+    std::shared_ptr< struct SortedCallstackReport >     m_SelectedSortedCallstackReport;
```

Comment:
Let's add a getter method instead, perhaps a 'bool HasCallstacks() const' since it seems we only care about whether the presence of callstacks?

Reply:
Addressed the issue. We are now using a HasCallstacks method.